modules/salt/pages/salt-states.adoc

```diff
@@ -164,3 +165,58 @@ Apply the highstate to all minions in the group:
 ----
 salt -I 'group_ids:GID' state.apply
 ----
+
+
+
+[[salt.pillars.setrpmendpoint]]
+== Use pillars to set the RPM endpoint
+
+By default, {productname} assumes that the {productname} server is the Salt master, and uses its single FQDN as the download endpoint for RPMs and repository metadata.
+However, there are some cases where you might like to use a different FQDN as the download endpoint.
+The most common example is if you need to use a proxy, or in environments with complicated networking requirements.
+
+To change the RPM endpoint, you can manually adjust three salt pillars:
+* [systemitem]``pkg_download_point_protocol``, defaults to [systemitem]``https``.
+* [systemitem]``pkg_download_point_host``, defaults to [systemitem]``fqdn``.
```

Comment:
Just to make it clear - this is not the string `fqdn` - it is the Fully Qualified Domain Name of the Server or Proxy.